ui/src/components/NodePageOverviewTab.js

```diff
@@ -19,13 +19,11 @@ import { intl } from '../translations/IntlGlobalProvider';
 
 const TabContentContainer = styled.div`
   overflow-y: auto;
-  height: 78vh;
+  height: 75vh;
```

Comment:
I think that we should compute the tab content here because likely `75vh` are not taking the full height of the page